src/components/ErrorBoundary/ErrorBoundary.js

```diff
@@ -2,7 +2,13 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
 
-import styles from './ErrorBoundary.module.scss';
+import {
+  BtnReportIssue,
```

Comment:
maybe we can avoid abbreviations like this one? maybe just `ReportButton` would be enough considering the scope where this button is defined